services/datamanager/builtin/builtin_capture_test.go

```diff
@@ -327,16 +323,16 @@ func waitForCaptureFilesToExceedNFiles(captureDir string, n int, logger logging.
 	start := time.Now()
 	for {
 		files := getAllFileInfos(captureDir)
-		nonEmptyFiles := 0
+		captureFiles := 0
 		for idx := range files {
-			if files[idx].Size() > int64(emptyFileBytesSize) {
+			if files[idx].Size() > int64(emptyFileBytesSize) && filepath.Ext(files[idx].Name()) == data.CompletedCaptureFileExt {
```

Comment:
we should only be asesrting on the contents of capture files, not .prog files. .prog files, by definition, are still in the progress of being written to so their contents can change at any time, resulting in flakey tests.